src/pvi/_format/screen.py

```diff
@@ -417,18 +417,30 @@ def generate_component_formatters(
                     c.children
                 ), "Header length does not match number of elements"
 
```

Comment:
I think here we should make a copy to use during this section, rather than resetting it at the end.
```suggestion
                header_bounds = component_bounds.copy()
```

Reply:
[and then](https://github.com/epics-containers/pvi/pull/63#discussion_r1401992222)